mlrun/feature_store/retrieval/dask_merger.py

```diff
@@ -50,10 +52,50 @@ def _generate_vector(
         # load dataframes
         feature_sets = []
         dfs = []
+        feature_sets_names = []
+        keys = []
+        all_columns = list()
+        self._parse_relations(feature_set_objects, feature_set_fields)
```

Comment:
add the keyword to the arguments:

```suggestion
        self._parse_relations(feature_set_objects=feature_set_objects, feature_set_fields=feature_set_fields)
```

It is much more robust to changes this way (if someone would have replace the order of things or whatever).